.travis.yml

```diff
@@ -14,6 +14,7 @@
 
 language: rust
 rust:
+  - 1.31.0
```

Comment:
The revert of this ended up in the wrong commit. You need to put it in the first commit.